streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -127,4 +138,50 @@ GeneralConfiguration getEventsToFhirConfig(String fileName) throws IOException {
 			return generalConfiguration;
 		}
 	}
+	
+	private String getUuid(String parentTable, String parentForeignKey, String childPrimaryKeyValue, Map payload) {
+		Connection conn = null;
+		Statement stmt = null;
+		final String USER_NAME = "root";
+		final String PASS_WORD = "debezium";
+		String uuidResultFromSql = null;
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
+			
+			//Connecting to openmrs database
+			conn = DriverManager.getConnection(DB_URL, USER_NAME, PASS_WORD);
```

Comment:
Let's set up and use a connection pool, e.g., HikariCP or C3P0; the latter is used in the batch pipeline so it would be nice to refactor the common pieces and move them to `common`.